english/new-movie/new-movie.md

```diff
@@ -0,0 +1,27 @@
+Scratched on the wall you read...
```

Comment:
Ah, the broken link made this unreachable, so I think this is fixed in the update.